styles.css

```diff
@@ -0,0 +1,265 @@
+
+* {
+    margin: 0px;
+    list-style-type: none;
+}
+
+
+
+
+a:visited {
+    color:white;
+    text-decoration: none;
+    list-style-type: none;
+
+}
+
+#footer {
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    height: 2.5rem;           
+    background-color: aliceblue;
+    opacity: .50;
+}
+
+
+
+#main body { 
+    background: url("file:///Users/chan/Developer/projects/personal-portfolio-site/images/portfolio_background.jpeg") no-repeat center center fixed; 
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+}
+
+
+
+
+
+
+
+nav img {
+    display: fixed;
+    opacity: .8;
+    border-radius: 50%;
+}
+
+
+
+.shape1 {
+    
+    display: block;
+    padding-left: 39%;
+    margin-top: 5%;
+    width: 50%;
+    height: 20rem;
+    width: 20rem;
+    position: fixed
+
+    
+}
+
+.shape1 img {
+    height: 18rem;
+    width: 18rem;
+}
+
+#transform:hover img {
+    display: none;
+    height: 20rem;
+    width: 20rem;
```

Comment:
I think the `.shape1` selector is already handling the height and width so you could probably take out those attributes from here.